app/src/main/java/org/mifos/mobilebanking/ui/fragments/RecentTransactionsFragment.java

```diff
@@ -147,9 +146,13 @@ public void onRefresh() {
         recentTransactionsPresenter.loadRecentTransactions(false, 0);
     }
 
+    /**
+     * Shows a Toast
```

Comment:
Change this to 'Shows a Toast when no more Recent Transactions are available

Reply:
@luckyman20 I don't think this is required is required. Since it's a generic method. We can call it anywhere to show a message.

Reply:
@miPlodder Ahh.. I see.